### ISSUE TITLE
Ktor 2.0 support

### ISSUE THREAD

**Issue by joreilly** [aPureBase/KGraphQL#175]
I realise Ktor 2.0 is still in eap right now but just curious if there's know changes needed/planned to support it?  I tried to update a project I have here to use Ktor 2.0 (using v0.17.14 of the library) but getting following

```
Type mismatch.
Required:
RouteScopedPlugin<TypeVariable(B), TypeVariable(F)>
Found:
GraphQL.Feature
```

**Comment by joreilly** [comment]
Just fyi that Ktor 2.0 is beta now so likely will see more projects starting to use it.

**Comment by sangeetds** [comment]
Need to change types, would like to undertake this change.

**Comment by NathanPB** [comment]
it makes me pretty worried because @jeggy seems not to be around to address this change

![image](https://user-images.githubusercontent.com/18128642/149808998-110f1355-e7f0-45aa-96e3-101283203123.png)

**Comment by sangeetds** [comment]
@NathanPB this is quite worrying :/

**Comment by sangeetds** [comment]
@NathanPB check again, @jeggy is still active I guess.

**Comment by NathanPB** [comment]
> @NathanPB check again, @jeggy is still active I guess.

Yes, I was getting worried because he used to be very active here and suddenly disappeared

**Comment by ronjunevaldoz** [comment]
is this still active?

**Comment by berserkore** [comment]
Hey People,

If you are looking to support this library with Ktor 2.0, I made a comment and temporary fix to support Ktor 2.0 until this library get revived.

https://github.com/aPureBase/KGraphQL/issues/185

Cheers

**Comment by sangeetds** [comment]
Hey @berserkore @ronjunevaldoz @NathanPB @joreilly and everyone else. 
I had already raised a PR #179 for fixing this but I guess the owner is currently inactive now. 
I have [forked](https://github.com/sangeetds/KGraphQL) this library where the particular commit is there and I have tested it for very basic cases and was seem to be working. Didn't get much time before to properly maintain it but I think from now I can maintain it since I have a use for it in my personal project too. Would love to get feedback and will plan to actively maintain it soon.

**Comment by jeggy** [comment]
Hi all. Sorry for being inactive for such a long time. 

I will start being active in this community again and I have now opened GitHub discussions and opened a discussion about getting more maintainers on this project (#187). 

We want to keep this repository active and maintain the [maven repository](https://search.maven.org/artifact/com.apurebase/kgraphql) and the https://KGraphQL.io/ website.
